drivers/sensor/CMakeLists.txt

```diff
@@ -38,5 +38,6 @@ add_subdirectory_ifdef(CONFIG_TMP007		tmp007)
 add_subdirectory_ifdef(CONFIG_TMP112		tmp112)
 add_subdirectory_ifdef(CONFIG_VL53L0X		vl53l0x)
 add_subdirectory_ifdef(CONFIG_LSM303DLHC_ACCEL	lsm303dlhc_accel)
+add_subdirectory_ifdef(CONFIG_LSM303DLHC_MAGN	lsm303dlhc_magneto)
```

Comment:
Usually the short word 'magn' is used to indicate magnetometer and not 'magneto'.
I suggest to use always 'magn'. If you agree to this rule, pls apply it everywhere in the commit.